.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+npm run fix:tables
+npm run test:links
+npm run test:build
+git update-index --again
```

Comment:
Note: This adds changed files (by fix:tables) to the current commit

Reply:
Furthermore, we might not need to run test:build, since it would be called on every commit

Reply:
Never remove runs from ci that are in precommit, precommit is not gauranteed in any way (if you don't run npm install for example)

Reply:
yeah i meant rather removing it from the precommit than from the ci